### PR TITLE
[deploy] Fix Image Overflow in Comment Preview - #4382

### LINKED ISSUE #4382: Images in comment preview are not scaled down to container width on mobile
**Describe the bug**
The image when uploaded through the mobile site/PWA isn't scaled down to fit the comment's container when in `preview` mode.

There is an earlier issue which #3264 which was marked as a duplicate of #3085 which was fixed by #3086 but from what I can see, that only tackles the image upload button's issue.

**To Reproduce**
1. Open any article on DEV, say [this](https://dev.to/wrldwzrd89/hacktoberfest-has-been-an-unmitigated-disaster-for-me-thus-far-mhl)
2. Go to comments, upload a large sized image. (In my case I just added a screenshot)
3. Click on the `preview` button.
4. The image will overflow.

**Expected behavior**
The image scales down to the preview container's size as it does when the comment is posted.

**Screenshots**
image

**Desktop (please complete the following information):**

- OS: N/A
- Browser: N/A
- Version: N/A

**Smartphone (please complete the following information):**

- Device: Nokia 6.1
- OS: Android 9
- Browser: Chrome
- Version: 77.0.3865.116

**Additional context**
The image is correctly sized when posted as the CSS for class `single-comment-node` adds CSS for the image to be `max-width: 100%`. A similar CSS solution should be viable for the preview.

### PR DESCRIPTION
**Pull request by Amorpheuz**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
- Enforces `max-width:100%` on images in `.comment-preview-div` to prevent them from overflowing their container.

## Related Tickets & Documents
Fixes #4382 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## Screenshots of bug fix

**Before:**
Screen Shot 2019-10-18 at 18 43 14

**After:**
Screen Shot 2019-10-18 at 18 42 59

## [optional] What gif best describes this PR or how it makes you feel?

get shrunk

### CODE REVIEW

File: app/assets/stylesheets/comments.scss [hunk 1]
```diff
@@ -345,6 +345,9 @@ a.header-link {
       code {
         word-wrap: break-word;
       }
+      .article-body-image-wrapper img{
```
Comment (jacobherrington) [inline app/assets/stylesheets/comments.scss]: Would you mind adding some whitespace after `img`?

I know we don't have the best formatting in our CSS, but it's probably a good idea to work on it as we go forward. :smile:

Comment (Amorpheuz) [reply app/assets/stylesheets/comments.scss]: Sure! Also regarding the formatting thing, maybe the core maintainers can decide on a tool to enforce conventions too, like Prettier or Stylelint and take benefit of the ongoing Hacktoberfest with refactoring PRs like the Js, Jsx and ruby files have been.

Comment (Amorpheuz) [reply app/assets/stylesheets/comments.scss]: I will get back to you with a screenshot in some time, busy for a few days. I had to wipe out my Linux install again cause an update messed with windows and some of my volumes with important data went missing. >.<

**Comment by Amorpheuz** [comment]
Uploaded Screenshots of the change @jacobherrington! (Bless WSL) ❤️

**Pull request merged**